### ISSUE TITLE
OSA Meeting Friday March 5th 2021

### ISSUE THREAD

**Issue by mattodd** [opensourceantibiotics/Series-2-Diarylimidazoles#57]
_Draft until this text removed_

Meeting March 5th 2021 at 2pm UK time at https://ucl.zoom.us/j/92800004715. This page follows on from #56.

Recording is [here].

On the call: 

No meeting was held on Feb 26th.

1. **Screening**

a) _Potency_
Results are in for the latest round of screening! 
[Lori's N-alkylated compounds](https://github.com/opensourceantibiotics/Series-2-Diarylimidazoles/issues/48):
![image](https://user-images.githubusercontent.com/53487492/108506218-5ae99480-72b0-11eb-83a0-8311f1cb837c.png)

[MSSA Results](
https://github.com/opensourceantibiotics/Series-2-Diarylimidazoles/issues/55)
![image](https://user-images.githubusercontent.com/53487492/108506286-72288200-72b0-11eb-90ca-b64d6f5557ec.png)

b) _Human Cytotox_
Underway in the lab of Andreas Schatzlein. This is also to test the metal-binding potential of the 2-pyridyl system. Updates will be at issue #3 .

Alex to present results!

Some discussion of the tox data for NEU compounds has taken place at #3. 

2. **Synthetic Chemistry**
Update from @danaklug 

3. **_in vitro_ PK**
Hypha have shipped their scaled-up material, due to arrive at UCL on March 2nd.

4. **Community updates**
a) Newsletters. Second newsletter has been sent and is available [here](https://github.com/opensourceantibiotics/murligase/issues/37).

5. **Mechanism of Action**
Results received via email and have been shared [here](https://github.com/opensourceantibiotics/Series-2-Diarylimidazoles/tree/master/Mechanism%20of%20Action/2020%20Initial%20UNC%20Data/MRSA%20Project). Discussion at #15 and #53.

**Action** on @edwintse to ensure the data are also installed on Labarchives and linked to Github, to ensure we don't lose such crucial files.

**Action**: Compile results of literature searches on targets of interest (mquo1, purD) and whether there are biochemical assays available.

6. **ELN**
@danaklug or @edwintse to try again (**Action**) Labarchives' molecule sketching facility to see if the ELN page is Google-able. i.e. sketch a molecule from OSA using the drawing facility, and check each day to see if it is indexed. ELN pages on which strings have been manually pasted are indexed (@mattodd checked this). Can we do away with manual strings pasting? Separately, @mattodd found that the OSA molecule spreadsheet is not indexed. @drc007 said he would check this out (**Action**).

AOB

- (ongoing -> install in wiki as a way of working) @mattodd also reminded people to keep an eye on sites such as Enamine, Manifold and Mcule, to check whether any molecules might be commercially available. We should always try to make the challenging molecules in the lab, and buy the others. Action on @edwintse to include this in wiki somewhere logical.

**Actions**
- [ ]  ACTION ITEM: Dana and Ed to look into whether manual strings are still needed in Labarchives, or whether molecule sketching facility is sufficient.
- [ ]  ACTION ITEM: Ed to outline on wiki how we should keep in mind searches for commercially-available molecules from various suppliers, in parallel to the in-house synthetic chemistry.
- [ ]  ACTION ITEM: Chris to look into why the OSA molecule sheet is not Google-indexed.
- [ ]  ACTION ITEM: This meeting to be posted on YouTube.

**Comment by MFernflower** [comment]
![takeda_paper](https://user-images.githubusercontent.com/3164942/109739841-3dfa6e80-7b98-11eb-8165-3b9bd25bcdb4.png)

**Comment by danaklug** [comment]
Chemistry update slide:
![Chemistry Updates](https://user-images.githubusercontent.com/53487492/110131776-6d8cbf00-7dc2-11eb-8c5a-102225673227.png)

**Comment by mattodd** [comment]
Hey everyone @flavioemery has just joined Github. I'll add him to the team. Flavio, these so-called "Issues" are where we raise items for discussion and share data. Ultimately they are all meant to be closed when the discussion is over and the data folded into the wiki (link above). The link to the sheet of available molecules can be found on [this page](https://github.com/opensourceantibiotics/Series-2-Diarylimidazoles/wiki/Submissions%2C-Resources%2C-and-Data), together with some basic rules of operation. Note that the series we're talking about is Series 2. There is a Series 1 targeted at the mur ligases where the structures are very different.

**Comment by MFernflower** [comment]
@mattodd @danaklug @loriferrins 

I think you may all find this interesting:

![Scheme-77-One-pot-two-step-process-for-the-synthesis-of-imidazopyrazine-3-8-diamines-as](https://user-images.githubusercontent.com/3164942/110188495-e05d6080-7de9-11eb-955b-2478714e8014.png)

10.1016/j.tet.2014.10.032

**Comment by mattodd** [comment]
Hi @flavioemery - in advance of our next OSA meeting, tomorrow, I wonder whether we could consult on which compounds we might want to exchange? All our compounds in this project are (I hope) in the [Master List](https://docs.google.com/spreadsheets/d/168-a1_l51Nfbms67eG8zU8p-EhEtEO26FUzRInbu7fY/edit?usp=sharing). Is it easy for you to do a search of those molecules to see if there is something useful? Similarly, I wonder if there is a way to find out which compounds are most similar to the structures we're interested in? The 6 compounds up the top of this post, from Lori, are probably the ones we'd like to base any future structures on, particularly structures that possess a chelating motif. Is there a way we can search for structures like this that you might have to hand? It would be very useful if someone were able to do a Tanimoto-style similarity search on a list of strings from Flavio, if that could be obtained. @danaklug @edwintse @drc007

**Comment by flavioemery** [comment]
Hi @mattodd and colleagues. Here you can find some representative compounds read to exchange. There are two lists (sorry for that) without those pyridine-substituted, because we don`t have them ready to ship. But, I am working on preparing them for you. I kept the information about the antiparasitic activity, just in case. I have just included those similar to Lori's and other's compounds. Unfortunately, missing chelating groups. But, we are definitely able to prepare them easily. Now, I am more familiar with github, and will post some scaffolds ideas we can apply to this project.  

[List 1.xlsx](https://github.com/opensourceantibiotics/Series-2-Diarylimidazoles/files/6125174/List.1.xlsx)

**Comment by flavioemery** [comment]
@mattodd @danaklug @loriferrins @edwintse @mcs07 @MFernflower  Here are some scaffolds from my lab that could be discussed. I have students working in this chemistry and they can try to synthesize some pyridinyl-substituted to introduce a chelating group to the scaffolds. 

![Flavio scaffolds](https://user-images.githubusercontent.com/80112736/110841605-de315100-8284-11eb-9796-fe817bfb19fb.png)
